src/main/java/ru/leo/models/grpc/ModelServiceGrpcImpl.java

```diff
@@ -0,0 +1,62 @@
+package ru.leo.models.grpc;
+
+import static ru.leo.models.grpc.GrpcUtils.req;
+
+import com.google.inject.Inject;
+import io.grpc.stub.StreamObserver;
+import java.util.List;
+import ru.leo.models.model.ModelService;
+
+public class ModelServiceGrpcImpl extends ModelsServiceGrpc.ModelsServiceImplBase {
+    private final ModelService modelService;
+
+    @Inject
+    public ModelServiceGrpcImpl(ModelService modelService) {
+        this.modelService = modelService;
+    }
+
+    @Override
+    public void loadModel(LoadModelRequest request, StreamObserver<LoadModelResponse> responseObserver) {
+        req("loadModel", request, responseObserver,
+            r -> {
+                modelService.loadModel(r.getModelName());
+                return LoadModelResponse.getDefaultInstance();
+            }
+        );
+    }
+
+    @Override
+    public void modelList(ModelListRequest request, StreamObserver<ModelListResponse> responseObserver) {
+        req("modelList", request, responseObserver,
+            r -> ModelListResponse.newBuilder()
+                .addAllModelNames(
+                    modelService.getModels()
+                ).build()
```

Comment:
```suggestion
                .addAllModelNames(modelService.getModels())
                .build()
```

Максимально вкусовщина, так что просто наброс